src/index.ts

```diff
@@ -19,7 +19,8 @@ export {
   matchRoute,
   generatePath,
   createLegacyHistory,
-  getRouteContext,
+  createRouterContext,
+  findRouterContext,
```

Comment:
can we keep the `getRouteContext` export and add a console.warn suggesting that consumers swap to `findRouterContext`? I think it's better to keep the API backward compatible as much as possible.

Reply:
I'm happy to write a breaking api / migration doc in the changelog, but given we are still v0.x, maintaining it right now would mean it will never be changed. Is it used in Bitbucket? Jira does not have usages